test/server/onListening-option.test.js

```diff
@@ -60,12 +74,71 @@ describe('onListening option', () => {
       );
     });
 
-    afterAll(testServer.close);
+    afterAll((done) => {
+      testServer.close(() => {
+        unlink(socketPath, done);
+      });
+    });
 
-    it('should run onListening callback', (done) => {
+    it('should run onListening callback, providing server without error', () => {
+      expect(resultingErr == null).toBeTruthy();
```

Comment:
please add the comment `// the err could be null or undefined`

Reply:
I did this, just the change isn't visible in this diff